yt/visualization/volume_rendering/scene.py

```diff
@@ -216,13 +226,13 @@ def render(self, camera=None):
         self._last_render = bmp
         return bmp
 
-    def save(self, fname=None, sigma_clip=None):
+    def save(self, fname=None, sigma_clip=None, render=True):
         r"""Saves the most recently rendered image of the Scene to disk.
```

Comment:
This line should be updated, I think it has become a bit contradictory with behavior you're aiming for.